### ISSUE TITLE
Run handtracking examples with leap motion

### ISSUE THREAD

**Issue by LuisMerinoP** [microsoft/MixedRealityToolkit-Unity#10501]
## Describe the issue
I followed [this](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/supported-devices/leap-motion-mrtk?view=mrtkunity-2021-05) reasource to run the examples and the project does not compile

## Feature area
What's incorrect? What's missing?
Following this resource the project does not compile

## Existing doc link
[https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/supported-devices/leap-motion-mrtk?view=mrtkunity-2021-05](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/supported-devices/leap-motion-mrtk?view=mrtkunity-2021-05)

## Additional context
I tried switching platform to universal windows platform, and tried both widows mixed reality and OpenXR 
![image](https://user-images.githubusercontent.com/49386405/158349558-20013bf5-5cf1-4b4d-9136-b3b5fd1ddc49.png)

Packages in project:

![image](https://user-images.githubusercontent.com/49386405/158349653-91bc2e59-7031-4664-b289-b56deb493caf.png)

Compilation errors:
![image](https://user-images.githubusercontent.com/49386405/158350121-fd892f5b-ab65-4f86-84ca-589476b5be74.png)

The error appears when trying to integrate leap motion into unity modules trying to find a Version.txt file in a path that does not make sense.

![image](https://user-images.githubusercontent.com/49386405/158356957-ddfb3ac1-c63e-4900-91b2-97d569eb62c5.png)

unity version 2020.3.11f

I would be glad to know if everything is supposed to work with the latest software and unitypackages versions or if there is any adviced version advice for downgrade to make it work.
Thanks for any comment or suggestion.

**Comment by LuisMerinoP** [comment]
I tried with UPM, with latest version in the package manager 4.8.0. I think I also tried importing the unitypackages myself which was I think 5.4 and could not make it work either, not a trustworthy trial because I was removing and installing everything back and forth.
What is the latest paired versions of the gemini software and the packages where the hand traking scenes work, please? Thanks a lot in advanced

**Comment by LuisMerinoP** [comment]
I also tried with Ultraleap Unity Modules 4.9.1, as soon as the package is imported and modules are integrated (Mixed Reality Toolkit > Leap Motion > Integrate Leap Motion Unity Modules) compilation errors are thrown.
![image](https://user-images.githubusercontent.com/49386405/158682895-6a5cf3cb-1a12-4ec7-96eb-fea32e304eb3.png)


Any advice to run the HandTrackingScene examples would be much appreciated.

**Comment by Zee2** [comment]
Hi, just a clarification:

When you build a new, blank project with MRTK and Ultraleap support, it functions as expected, but when opening the existing MRTK project, it doesn't work?

In addition, does this issue still happen when using older versions of Gemini?

**Comment by LuisMerinoP** [comment]
Thanks for your interest, I solved the issue following thies steps

1.-Ensure the Microsoft.MixedReality.Toolkit.Examples feature has been added to your Unity project via the Mixed Reality Feature Tool.
2.-Open the Package Manager in Unity and find the "Mixed Reality Toolkit Examples" package in the list and click to select it.
3.-Open the "Samples" dropdown and find "Demos - Hand Tracking". Click Import.
There should be no compilations errors.
4.-Open the "HandInteractionExamples" scene.
5.-Change the profile for this scene to "DefaultLeapMotionConfigurationProfile".
6.-Press play and see your hands in the scene. There should be no runtime errors.
